sdk/storage/storage-queue/test/utils/recorder.ts

```diff
@@ -0,0 +1,417 @@
+import fs from "fs-extra";
+import nise from "nise";
+import queryString from "query-string";
+import { getUniqueName, isBrowser } from "../utils";
+import { blobToString } from "./index.browser";
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../.env" });
+
+let nock: any;
+if (!isBrowser()) {
+  nock = require("nock");
+}
+
+const env = isBrowser() ? (window as any).__env__ : process.env;
+const isRecording = env.TEST_MODE === "record";
+const isPlayingBack = env.TEST_MODE === "playback";
+
+if (isPlayingBack) {
+  // Providing dummy values to avoid the error
+  env.ACCOUNT_NAME = "fakestorageaccount";
+  env.ACCOUNT_KEY = "aaaaa";
+  env.ACCOUNT_SAS = "aaaaa";
+}
+
+/**
+ * Additional layer of security to avoid unintended/accidental occurances of secrets in the recordings
+ * */
+export function filterSecrets(recording: string): string {
+  return recording
+    .replace(env.ACCOUNT_NAME, "fakestorageaccount")
```

Comment:
[Only the first occurrence will be replaced like this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace). Make it a regular expression so we can replace all occurrences (for ACCOUNT_KEY and ACCOUNT_SAS as well).